sia-gpu-miner.c

```diff
@@ -71,21 +78,24 @@ double grindNonces(size_t items_per_iter, int cycles_per_iter) {
 		}
 		target_corrupt_flag = 1;
 		printf("Received corrupt target from Sia\n");
-		printf("Usually this resolves itself within a minute or so\n");
-		printf("If it happens frequently trying increasing seconds per iteration\n");
-		printf("e.g. \"./gpu-miner -s 3 -c 200\"\n");
+		// API call may have returned an error string
+		printf("Possible error text:\n");
+		for (i = 0; i < 32 && target[++i];) {
+			printf("%c", target[i-1]);
+		}
+		printf("\n");
```

Comment:
This is okay for now, but, as I commented on the commit that this appeared in, there's a more correct way to check for errors when making a curl.